src/components/HomePage.js

```diff
@@ -2,35 +2,90 @@ import styled from 'styled-components';
 import Button from 'react-bootstrap/Button';
 import Col from 'react-bootstrap/Col';
 import { useNavigate } from 'react-router-dom';
+import { CardGroup, Card} from 'react-bootstrap';
+
+
+
+import "./style.css";
 
-// For now, very basic home page with a button that leads to the module selection page
 const HomePage = () => {
     // useNavigate hook will be used to programmatically route the user based on their actions
     const navigate = useNavigate();
     return (
-        <Wrap>
-            <HeadingWrap>
-                Home Page
-            </HeadingWrap>
-            <Button
-                as={Col}
-                variant="primary"
-                onClick={() => {
-                    // When this button is clicked, route the user to the module selection page
-                    navigate('/modules');
-                }}
-            >
-                Module Selection Page
-            </Button>
+        
+        <Wrap>     
+            <Heading>
+               <strong>&nbsp;&nbsp;&nbsp;&nbsp;NeoCirc</strong>
+            </Heading>
+            <SubHeading2 width="95%">
+            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+            <i>An educational platform leveraging e-learning to promote proficiency in medical procedures.</i>
+            </SubHeading2>
+            <div style={{
+                alignItems: 'center',
+                justifyContent: 'center',
+                padding: '105px'
+            }}>
```

Comment:
So this works fine, but might be nice to wrap it in a styled-components component to keep it consistent